src/models/forms/neuburgForm.js

```diff
@@ -1,6 +1,6 @@
 // @flow
 
-const VERSION = 1.0
+const VERSION = '1'
```

Comment:
I think we need a custom form for every version so a number would be simpler. What do you think @Schedulaar ?

Reply:
But then the version in the form can't be of type string:)

Reply:
yep, I just wanted to note that the type of the value and the type differ